scripts_limit/detailRT_fig3.py

```diff
@@ -0,0 +1,162 @@
+import pandas as pd
+import sys
+
+import matplotlib.pyplot as plt
+
+# user-defined variables
+#################################################################################
+if len(sys.argv) != 9:
+    print(
+        "Usage: python detailRT_fig3.py output_name title_name x1 x2 timestamp_offset collectls hosts"
+    )
+    sys.exit(1)
+
+output_name = sys.argv[1]
+title_name = sys.argv[2]
+x1 = sys.argv[3]
+x2 = sys.argv[4]
+timestamp_offset = sys.argv[5]
+collectls = sys.argv[6].split(",")
+collectlNW = sys.argv[7].split(",")
+hosts = sys.argv[8].split(",")
+
+num_cores = 8
+
+collectls_cols = [
```

Comment:
What do these number mean? Are they hard code?

Reply:
These are the columns to be graphed that were hardcoded in the original `detailRT_fig3.gnuplot` code